java/ql/src/semmle/code/java/PrintAst.ql

```diff
@@ -16,5 +16,6 @@ class PrintAstConfigurationOverride extends PrintAstConfiguration {
   /**
    * TWEAK THIS PREDICATE AS NEEDED.
    */
-  override predicate shouldPrint(Element e, Location l) { super.shouldPrint(e, l) }
+  override predicate shouldPrint(Element e, Location l) { super.shouldPrint(e, l) and 
+    not l.getFile().getBaseName().matches("SystemGetPropertyCall.java") }
```

Comment:
Accidentally committed?

Reply:
Nope, this is intentional. Do you want me to just update the test result?

Reply:
Yes, please update the test result instead.

Reply:
This should be resolved now.